_config.yml

```diff
@@ -9,6 +9,11 @@ banner: "images/banner.jpg"
 excerpt_link: Read More
 fancybox: true
 
+# Footer
+copyright: |-
+  <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></br>
```

Comment:
IMHO, we should not set a default license.
I prefer to write a link for help like below instead of a default license.

> https://choosealicense.com/
> https://creativecommons.org/choose/

```
# Footer
copyright:
   # Need help choosing? Please see...
   # https://choosealicense.com/
   # https://creativecommons.org/choose/
```

Reply:
Yes, that's true.

I'm trying to let user know this value can render `html`. 
So, how do you suggest to do that?

Reply:
@RammusXu 

Hmm...
how about like below?

```
# Footer
copyright:
   # Need help choosing? Please see...
   # https://choosealicense.com/
   # https://creativecommons.org/choose/
# You can use html like below...
#copyright: |-
#  <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/"><img alt="Creative Commons License" #style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" /></a></br>
#  All website licensed under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" target="_blank">CC BY-NC-#ND 4.0</a></br>
```